test/nextcloud/1.1.0/templates/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       initContainers:
         - name: init-postgresdb
           image: busybox:latest
-          command: ['sh', '-c', "until nslookup {{ template "nextcloud.fullname" . }}-postgres; do echo waiting for postgres; sleep 2; done"]
+          command: ['sh', '-c', "until nc -w 5 -vz {{ template "nextcloud.fullname" . }}-postgres 5432; do echo waiting for postgres; sleep 2; done"]
```

Comment:
Apparently this won't work based on @themylogin 's comment on the previous PR that you just merged.

Reply:
It would work because k8s will restart the container if it fails but still @themylogin is correct and it's better to use postgres container in this case.